src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzResultsContentPanel.java

```diff
@@ -131,7 +163,7 @@ public void itemStateChanged(ItemEvent e) {
         mainPanel.add(fuzzResultTableScrollPane);
 
         add(toolbar, BorderLayout.PAGE_START);
-        add(mainPanel, BorderLayout.CENTER);
+        add(mainPanel, BorderLayout.CENTER); 

```

Comment:
Oops, I forgot to undo that. Also, now that I notice, what does it mean when it says in the guidelines that the Strings used should be internationalized? I understand they have to be translated, but do I do anything differently coding-wise?